src/main/java/org/example/homework9/MobilePhone.java

```diff
@@ -0,0 +1,86 @@
+package org.example.homework9;
+
+import java.util.ArrayList;
+
+public class MobilePhone {
+    private String myNumber;
+    private ArrayList<Contact> myContacts;
+
+    public MobilePhone(String myNumber, ArrayList<Contact> myContacts) {
+        this.myNumber = myNumber;
+        this.myContacts = myContacts;
+    }
+
+    public boolean addNewContact(Contact contact) {
+        int index = findContact(contact);
+        if (index >= 0) {
+            System.out.println("Such contact already exists.");
+            return false;
+        } else {
+            myContacts.add(contact);
+            System.out.println("Contact '" + contact.getName() + "' added successfully.");
+            return true;
+        }
+    }
+
+    public boolean updateContact(Contact oldContact, Contact newContact) {
+        int index = findContact(oldContact);
+        if (index >= 0) {
+            myContacts.set(index, newContact);
+            System.out.println("Contact '" + oldContact.getName() + "' was replaced by contact '" + newContact.getName() + "'.");
+            return true;
+        } else {
+            System.out.println("No such contact.");
+            return false;
+        }
+    }
+
+    public boolean removeContact(Contact contact) {
+        int index = findContact(contact);
+        if (index >= 0) {
+            myContacts.remove(contact);
+            System.out.println("Contact '" + contact.getName() + "' was removed successfully.");
+            return true;
+        } else {
```

Comment:
```suggestion
            System.out.println("No such contact.");
            return false;
        }
```
Minor: Just as an option. When you have a method with return statement it is not mandatory to use `else`, you can just finish your `if {... return true;}` logical branch and code from else branch will be executed even without using `else` statement. It looks better and easier to read